### ISSUE TITLE
Sketch with huge canvas background layer - Arguments to path.resolve must be strings

### ISSUE THREAD

**Issue by pierrelucgagnon** [avocode/avocode#502]
@rychlis as requested

IMPORTANT: Please try to describe how did it happen.

-----
Sketch Exporter 2.1.3 running in Sketch 3.3.3
```
TypeError: Arguments to path.resolve must be strings
    at Object.exports.resolve (path.js:313:15)
    at /Users/plgagnon/Library/Containers/com.bohemiancoding.sketch3/Data/…/avocode-exporter/lib/image-exporter.js:1:297
    at Object.oncomplete (fs.js:107:15)
```

**Comment by rychlis** [comment]
Thanks a lot, is it possible for you to share the design with us at team@avocode.com please?

**Comment by rychlis** [comment]
Oh i can see you already shared it, thanks!

**Comment by pierrelucgagnon** [comment]
no problem, hope this help.

**Comment by rychlis** [comment]
The issue was caused by a huge layer in the document that's only used to replace the canvas color. Export of such layer would fail (the layer was 50967x40146 in size) So the workaround for now is to remove such layer and we will try to fix the extension by ignoring such layers or exporting them in a different way

**Comment by rychlis** [comment]
This error is now #513 in the newer version of Sketch plugin (2.1.4+). We have released beta version of a new extension Sketch 3.0 which is not affected by this issue. See this guide to install and let me know if there are any problems: https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md
